github-templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+(brief description / PR title)
+
+Status: **Ready for Review** or **Open for Visibility**
+Reviewers: <reviewers here>
+
+JIRA: <link to applicable JIRA ticket>
+
+## Changes
+- <changes here>
+
+## Todos:
+- [ ] +1
+- [ ] Passes tests
+- [ ] Updated README
+- [ ] Updated CHANGELOG
+- [ ] <Other TODOs>
+
+### Feedback:
+_none so far_
```

Comment:
How often is this section used?  I don't know that I've _ever_ seen it used on Astro. We always use commenting per line or at the bottom of the Conversations view.